tests/test_linter/test_error.py

```diff
@@ -31,7 +31,7 @@ def test_to_string(test_issue, verbose, expected):
     Test the string formats for LinterError
     """
 
-    assert test_issue.to_string(verbose) == expected
+    assert str(test_issue.to_string(verbose)) == expected
```

Comment:
we should not accept a non-string response from to_string, so the test here is too forgiving by converting the returned value.

Reply:
so to_string was returning a rich Text and it was simpler to cast it to string. I have changed tthe name of the funciton to `to_rich` which makes more sense and aligned with other changes.